docker/cygnus-ngsi/Dockerfile

```diff
@@ -154,12 +154,33 @@ ENV CYGNUS_POSTGRESQL_ENABLE_CACHE ""
 # ENV CYGNUS_CARTO_USER_FILE
 # ENV CYGNUS_CARTO_KEY_FILE
 
+# Orion options
+ENV CYGNUS_ORION_ENABLE_ENCODING ""
+ENV CYGNUS_ORION_ENABLE_GROUPING ""
+ENV CYGNUS_ORION_ENABLE_NAME_MAPPINGS ""
+ENV CYGNUS_ORION_ENABLE_LOWERCASE ""
+ENV CYGNUS_ORION_HOST ""
+ENV CYGNUS_ORION_PORT ""
+ENV CYGNUS_ORION_SSL ""
+ENV CYGNUS_ORION_USER ""
+ENV CYGNUS_ORION_PASS ""
+ENV CYGNUS_ORION_KEYSTONE_HOST ""
+ENV CYGNUS_ORION_KEYSTONE_PORT ""
+ENV CYGNUS_ORION_KEYSTONE_SSL ""
+ENV CYGNUS_ORION_FIWARE ""
+ENV CYGNUS_ORION_FIWARE_PATH ""
+ENV CYGNUS_ORION_BATCH_SIZE ""
+ENV CYGNUS_ORION_BATCH_TIMEOUT ""
+ENV CYGNUS_ORION_BATCH_TTL ""
+ENV CYGNUS_ORION_BATCH_RETRY_INTERVALS ""
+ENV CYGNUS_ORION_ENABLE_CACHE ""
+
 
 # NOTE: Configure correctly GIT_URL_CYGNUS and GIT_REV_CYGNUS for each git branch/fork used
 ENV GIT_URL_CYGNUS "https://github.com/telefonicaid/fiware-cygnus.git"
 # ENV GIT_URL_CYGNUS "https://github.com/myuser/fiware-cygnus.git"
-ENV GIT_REV_CYGNUS "master"
```

Comment:
This should be fixed. I understand that "master" should be the right one.

Reply:
Fixed in a5a550e971e6179f2570362ec59ef17bb1475975